airbyte-integrations/connectors/source-hubspot/source_hubspot/streams.py

```diff
@@ -1374,6 +1377,62 @@ class ContactsAllBase(Stream):
     records_field = None
     filter_field = None
     filter_value = None
+    _state = {}
+    limit_field = "count"
```

Comment:
Nice catch. It was using `limit` before. The default value on the API side is 100 so this shouldn't change the behavior